config/cmake/MFEMConfig.cmake.in

```diff
@@ -11,7 +11,10 @@
 
 include(${CMAKE_CURRENT_LIST_DIR}/MFEMConfigVersion.cmake)
 
+set(MFEM_MPIRUN_CMD @MFEM_MPIRUN_CMD@)
```

Comment:
Perhaps splitting them ( like cmake ) will be the most flexible solution:
MFEM_MPIEXEC
MFEM_MPIEXEC_NUMPROC_FLAG

@v-dobrev - have a preference?